pom.xml

```diff
@@ -92,16 +92,11 @@
             <version>${junixsocket.version}</version>
             <optional>true</optional>
         </dependency>
-        <dependency>
-            <groupId>com.kohlschutter.junixsocket</groupId>
-            <artifactId>junixsocket-native-common</artifactId>
-            <version>${junixsocket.version}</version>
-            <optional>true</optional>
-        </dependency>
         <dependency>
             <groupId>com.kohlschutter</groupId>
             <artifactId>compiler-annotations</artifactId>
             <version>1.5.2</version>
+            <scope>provided</scope>
```

Comment:
Using `provided` scope combined with the `includeProvidedScope = false` setting in cyclonedx-maven-plugin prevents a dependency entry from being added to the CycloneDX SBOMs for this, since we only include this to avoid compiler warnings caused by the annotations inside junixsocket.